apps/designer/package.json

```diff
@@ -40,10 +42,17 @@
         "config": "workspace:*",
         "eslint": "7.32.0",
         "eslint-config-custom": "workspace:*",
+        "konva": "^8.3.13",
         "next-transpile-modules": "9.0.0",
         "postcss": "^8.4.16",
+        "react-konva": "^18.2.2",
+        "react-native": "^0.70.2",
+        "react-spring": "^9.5.5",
+        "react-zdog": "^1.0.11",
         "tailwindcss": "^3.1.8",
+        "three": "^0.145.0",
```

Comment:
why is this needed?

Reply:
Initially we had decided to use React-Spring, so it was a required dependency. Now it has been deleted from the package.json.